Doc/howto/argparse.rst

```diff
@@ -13,11 +13,15 @@ recommended command-line parsing module in the Python standard library.
 
 .. note::
 
-   There are two other modules that fulfill the same task, namely
-   :mod:`getopt` (an equivalent for ``getopt()`` from the C
-   language) and the deprecated :mod:`optparse`.
-   Note also that :mod:`argparse` is based on :mod:`optparse`,
-   and therefore very similar in terms of usage.
+   The standard library includes two other libraries directly related
+   to command-line parameter processing: the lower level :mod:`optparse`
+   module (which may require more code to configure for a given application,
+   but also allows an application to request behaviors that ``argparse``
+   doesn't support), and the very low level :mod:`getopt` (which specifically
+   serves as an equivalent to :c:func:`!getopt` from the C language).
```

Comment:
Python `getopt` provides behavior that combines the classic `getopt` and the GNU `getopt_long`, so the reference (I realize it was there in the original) is incomplete. And to be a little nitpicky, the ISO C standard says nothing about `getopt` (POSIX does, but only the single-char `getopt`) so From C could be mildly confusing. Maybe something like _(which specifically serves as an equivalent to the getopt family of functions familiar to C language programmers)_?

Reply:
Nice phrasing! I've used this in a couple of places (but with "available" instead of "familiar", since not all C programmers will be familiar with them, but they will almost always have them available)